public/js/deleteEvent.js

```diff
@@ -1,4 +1,4 @@
-async function deleteFormHandler(event) {
+async function deleteEventFormHandler(event) {
```

Comment:
I just posted a comment on Slack about this file but it if remains I have the same comment about the function definition:
const deleteEventFormHandler = async (event) => {